pkg/api/runner.go

```diff
@@ -90,6 +95,10 @@ func withSemaphore(fns []Request, maxConcurrency int, requestTimeout *time.Durat
 }
 
 func (r *Runner) Run(ctx context.Context) error {
+	if r.Logger != nil {
+		logger = *r.Logger
```

Comment:
What happens if you have multiple runners with different loggers?

Reply:
...you're right, I was thinking about this in the context of how our lambda uses it (single runner), but it's a separate, public library